quicktask/app/Model/Task.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $table = 'tasks';
+    protected $fillable =   [id, name];
```

Comment:
```suggestion
    protected $fillable = [
        'id',
        'name',
    ];
```
Viết như thế kia nó có chạy ko?